skycatalogs/utils/config_utils.py

```diff
@@ -138,7 +138,7 @@ def get_config_value(self, key_path, silent=False):
         path_items = key_path.split('/')
         d = self._cfg
         for i in path_items[:-1]:
-            if not i in d:
+            if not i in d.keys():
```

Comment:
I had to change it because the original `if not in d` didn't do the right thing.  d is not exactly a dict; it's a Config which delegates to dict.  I would have to write some more code to get it to handle the original syntax correctly. It didn't seem worth it.